packages/web/src/executors/package/package.impl.ts

```diff
@@ -39,6 +28,16 @@ import {
 import { deleteOutputDir } from '../../utils/delete-output-dir';
 import { runRollup } from './run-rollup';
 
+// These use require because the ES import isn't correct.
+const resolve = require('@rollup/plugin-node-resolve');
```

Comment:
Question: can we not use the import syntax for importing a default export here? e.g.:

```ts
import resolve from '@rollup/plugin-node-resolve';
```

Reply:
It depends on the es interopt option in tsconfig. At the very least we can probably do `import resolve = require('@rollup/plugin-node-resolve');`

Reply:
> > This looks awesome. Thanks so much. Will this work with pnpm? When I tried your gist, my `id` value was the full system path
> 
> Yes there isn't any different between how we handle npm, yarn, pnpm from a dependency detection perspective.

Then I am not sure what I am doing wrong but I am getting errors 
```sh
Error during bundle: Error: 'isEmpty' is not exported by node_modules/.pnpm/lodash@4.17.21/node_modules/lodash/fp.js
Bundle failed: ui-hooks-use-resize-observer
```

`isEmpty` is imported from lodash in this case. 

```json
{
  "root": "libs/ui",
  "sourceRoot": "libs/ui",
  "projectType": "library",
  "targets": {
    "build": {
      "executor": "@nrwl/web:package",
      "outputs": ["{options.outputPath}"],
      "options": {
        "outputPath": "dist/libs/ui/package",
        "tsConfig": "libs/ui/tsconfig.lib.json",
        "project": "libs/ui/package.json",
        "entryFile": "libs/ui/index.ts",
        "assets": [
          {
            "glob": "libs/ui/README.md",
            "input": ".",
            "output": "."
          }
        ]
      }
    },
    "release": {
      "executor": "@nrwl/workspace:run-commands",
      "options": {
        "commands": [
          {
            "command": "npx semantic-release"
          }
        ],
        "cwd": "libs/ui"
      }
    }
  },
  "tags": []
}

```